docs/digger/admin/managing-digger-logs-artifacts.asciidoc

```diff
@@ -0,0 +1,31 @@
+== Managing Digger Build Logs and Artifacts
+App builds started by Digger produce build logs and build artifacts.
+Over time, these will build up. There are a number of options to manage the buildup. If you are using Digger with RHMAP, these options can be specified
+by setting environment variables in the `millicore` deployment(???) configuration in
```

Comment:
@wei-lee @laurafitzgerald @matskiv 

q2 : re: enivornment variables. In RHMAP, these can be set in the milicore deployment. without RHMAP, where/how?

q3: i can copy this to our downstream repo @finp and modify it as follows:

App builds started by Digger produce build logs and build artifacts. Over time, these will build up. There are a number of options to manage the buildup. <insert or reference to guide how to set environment variables = q2>.Configuration options found in Jenkins along with associated Environment variables are detailed below. 

RHMAP property -> Environment variable